images/hive.yml

```diff
@@ -10,7 +10,7 @@ from:
   builder:
   - stream: rhel
   member: hadoop
-name: openshift/ose-metering-hive
```

Comment:
however I think @chancez just made it this recently and it should probably be `metering-hive` upstream too?

Reply:
The OKD was wrong. This should be left unchanged.